backend/chat/models.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+from config.settings import CHOICE_ROOM
+from django.contrib.auth import get_user_model
+from django.db import models
+from PIL import Image
+
+
+class Chat(models.Model):
+    """Chat table"""
+
+    title = models.CharField(max_length=70)
+    room = models.CharField(choices=CHOICE_ROOM, max_length=50)
+    img = models.ImageField(
+        null=True,
+        blank=True,
+        default="default-user-avatar.jpg",
+        upload_to="chat_img",
+    )
+    creator = models.ForeignKey(get_user_model(), null=True, on_delete=models.CASCADE)
+    description = models.TextField(blank=True, null=True)
```

Comment:
Django documentation says that it is better to avoid using [null](https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.Field.null) on string-based fields such as [CharField](https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.CharField) and [TextField](https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.TextField).

https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.Field.null

Reply:
But we discussed that users will have the option to delete their profiles. So when we delete the user, we need to set the creator field to null if we want to save his chats. Or should we delete his chats?

If the user does not want to write a description of the chat, then to set the field to an empty value, it must be that values blank=True, null=True